apis/apis.json

```diff
@@ -289,7 +289,7 @@
     "type": "other",
     "targetFrameworks": "netstandard2.0",
     "testTargetFrameworks": "netcoreapp2.1;net461",
-    "description": "Google Stackdriver Instrumentation Libraries for ASP.NET Core.",
+    "description": "Google Cloud Logging Instrumentation Libraries for ASP.NET Core.",
```

Comment:
Should be Logging, Trace and Error Reporting. For all Google.Cloud.Diagnostics.*

Reply:
Will fix this as a follow-up commit (in this PR still).